wdl2cwl/main.py

```diff
@@ -152,6 +163,29 @@ def get_expr_name_with_is_file_check(wdl_expr: WDL.Expr.Ident) -> str:
     return expr_name if not is_file else f"{expr_name}.path"
 
 
+def get_workflow_outputs(
+    outputs: WDL.Env.Bindings[WDL.Type.Base],
+) -> Iterator[Tuple[str, Union[cwl.OutputArraySchema, str], str]]:
+    """Return the name, CWL type, and source for a workflow's effective_outputs()."""
+    for item in outputs:
+        output_name = item.name
+        meta_name = item.info.expr.expr.name[::-1].replace(".", "/", 1)[::-1]
```

Comment:
This is really cool and it totally works.

Reply:
The only issue I see with this is in the case of imported tasks. This meta_name doesn't reflect the name of the task it is being imported from. If this is not included the cwltool raises unidentified outputsource error.

Reply:
> The only issue I see with this is in the case of imported tasks. This meta_name doesn't reflect the name of the task it is being imported from. If this is not included the cwltool raises unidentified outputsource error.

This is the same code we had before, just in a different location. Can you provide a link to a WDL file that causes an issue?

Reply:
It's needed to solve the validation problem as in the case of `align_and_count.wdl`

Reply:
But that's why I approved it. I can make the change to pass validate. This is clearly a better way to handle workflow inputs an I did before with the get_cwl_outputs in PR #156

Reply:
Okay. Just confirming that you want me to merge as-in :-)